src/styles/card.css

```diff
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  justify-content: space-evenly;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: var(--main-color);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  display: inline-block;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  color: var(--light-color);
+  font-size: 10px;
+  margin-bottom: 16px;
+  line-height: 14px;
+}
+
+.card__ratting {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__ratting-rewiev {
+  font-size: 10px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 12px;
+}
+
+.card__price-title {
+  display: inline-block;
+  color: var(--light-color);
+  line-height: 18px;
+}
+
+.card__price-sum {
+  display: inline-block;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  padding-bottom: 4px;
+}
+
+.card__button {
+  background-color: var(--blue-color);
+  width: 100%;
```

Comment:
write less code. it is already 100% because its display prop. is set to block, because its parent is a flex conntainer